README.md

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
This is a markdown file. Please take a look at the markdown cheatsheet to write the pieces of content you need to include.

e.g., 

```
# My Project

is...

# Wireframes

![Wireframe of About Me](wireframes/7420AboutMe.jpg)

This page has the following blocks of content...
```

Reply:
I'm confused by what you're asking me to fix, I thought I had done that correctly already. I see my images in the preview screen for Visual Studio Code (see screenshot attached). I also followed the video's steps to commit and push everything, or at least I think I did. Thanks! 
![2018-09-12](https://user-images.githubusercontent.com/42822113/45460408-51b17200-b6cb-11e8-89e0-fd11cc43ba0e.png)

Reply:
The current state of this README.md file includes HTML instead of what you have in your screenshot. Are there uncommitted changes that are not included here?

`git status` to check if you have modified files
`git add README.md` to add modified README.md file
`git commit -m "Updates README to include wireframes"`
`git push origin wireframes`

Reply:
Thank you. I guess for some reason I thought I had done that initially but hadn't. I think it should have been fixed now, but please let me know either way.